### PR TITLE
Adjust redirects

### PR DESCRIPTION
**Pull request by adegeo**

### CODE REVIEW

**Comment by adegeo** [comment]
@tdykstra

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -203,6 +203,30 @@
...
+        {
+            "source_path": "dotnet-desktop-guide/net/winforms/how-keyboard-input-works.md",
+            "redirect_url": "/dotnet/desktop/winforms/input-keyboard/overview?view=netdesktop-5.0"
+        },
+        {
+            "source_path": "dotnet-desktop-guide/net/winforms/how-mouse-input-works-in-windows-forms.md",
+            "redirect_url": "/dotnet/desktop/winforms/input-mouse/overview?view=netdesktop-5.0"
         }
     ]
 }
```
Comment (tdykstra) [inline .openpublishing.redirection.json:232]: missing end of file line break

**Review by tdykstra — APPROVED** [review]
LGTM

File: .openpublishing.redirection.json (hunk 1 above)
Comment (adegeo) [inline .openpublishing.redirection.json:232]:
```suggestion
}

```

**Pull request merged**